lib/rubocop/config.rb

```diff
@@ -23,6 +23,10 @@ def initialize(hash = {}, loaded_path = nil)
       super(hash)
     end
 
+    def to_s
+      @to_s ||= __getobj__.to_s
```

Comment:
`__getobj__` returns the `hash` object that was passed to `super` in the initializer, right?